ROMFS/px4fmu_common/init.d-posix/airframes/10043_sihsim_standard_vtol

```diff
@@ -91,12 +79,6 @@ param set-default MAV_TYPE 22
 # set SYS_HITL to 2 to start the SIH and avoid sensors startup
 # param set-default SYS_HITL 2
 
-# disable some checks to allow to fly:
-# - without real battery
-param set-default CBRK_SUPPLY_CHK 894281
```

Comment:
Thinking again about this: we may want to keep it for the SIH-on-FC, to be able to simulate a flight without having to plug a battery (e.g. for flight controllers that can be powered over USB).

Reply:
makes sense. added it again for SIH on FC (which is 1103, not 10043 here)